src/controller/CHIPDeviceController.cpp

```diff
@@ -408,6 +418,8 @@ DeviceCommissioner::DeviceCommissioner() :
 
 CHIP_ERROR DeviceCommissioner::Init(CommissionerInitParams params)
 {
+    VerifyOrReturnError(params.operationalCredentialsDelegate != nullptr, CHIP_ERROR_INVALID_ARGUMENT);
+    mOperationalCredentialsDelegate = params.operationalCredentialsDelegate;
```

Comment:
So mOperationalCredentialsDelegate is only used by DeviceCommissioner, right?  The member should move to that class...

Ideally it would also move out of ControllerInitParams, but that might be a breaking change.

Reply:
Yes, there a public function `GetOperationalCredentialsDelegate` in DeviceController class. Move mOperationalCredentialsDelegate to DeviceCommissioner class may also cause a breaking chane.....